### PR TITLE
Updating ws package

### PR DESCRIPTION
**Pull request by eliperelman**

### CODE REVIEW

**Comment by eliperelman** [comment]
@jonasfj would you mind testing this out, I don't have a docker installation, and I'd like to make sure the tests pass.

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "promise":			"6.1.0",
     "slugid": 			"1.0.3",
     "through2": 		"^2.0.0",
-    "ws": 			"1.0.1",
+    "ws": 			"1.1.1",
```
Comment (jonasfj) [inline package.json:19]: Any reason not to use `^1.1.0` down streamers should use npm-shrinkwrap.

**Review by jonasfj — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (eliperelman) [reply package.json:19]: Fine with me.

**Review by jonasfj — APPROVED** [review]

**Comment by jonasfj** [comment]
This works...

**Comment by jonasfj** [comment]
I'll do that change and publish, now that my editor is open :)

**Pull request merged**